util/log/log.go

```diff
@@ -224,14 +224,14 @@ func callerLocation() string {
 }
 
 func Debug(a ...interface{}) {
-	if Log.GetLevel() > debugLog {
+	if Log != nil && Log.GetLevel() > debugLog {
```

Comment:
Do we need to check if `Log != nil` at other places?

Reply:
log.Info, Warning, Error, Fatal all has this handled already (see `Output()` and `Outputf()`)